doc/source/tutorial/optimize.rst

```diff
@@ -2002,3 +2002,85 @@ For more MILP tutorials, see the Jupyter notebooks on SciPy Cookbooks:
 
 - `Compressed Sensing l1 program <https://nbviewer.org/github/scipy/scipy-cookbook/blob/main/ipython/LinearAndMixedIntegerLinearProgramming/compressed_sensing_milp_tutorial_1.ipynb>`_
 - `Compressed Sensing l0 program <https://nbviewer.org/github/scipy/scipy-cookbook/blob/main/ipython/LinearAndMixedIntegerLinearProgramming/compressed_sensing_milp_tutorial_2.ipynb>`_
+
+
+Parallel execution support
+---------------------------
+
+Some SciPy optimization methods, such as :func:`differential_evolution` offer
+parallelization through the use of a ``workers`` keyword. Each optimizer is slightly
+different in how parallelization is employed, but there are common characteristics
+in what can be provided to ``workers``. These commonalities are described below.
+
+If an int is supplied then a :class:`multiprocessing.Pool <multiprocessing.pool.Pool>` is
+created, with the object's :func:`map` method being used to evaluate solutions in
+parallel. With this approach it is mandatory that the objective function is pickleable.
+Lambda functions do not meet that requirement.
+
+::
+
+    >>> import numpy as np
+    >>> from scipy.optimize import rosen, differential_evolution, Bounds
+    >>> bnds = Bounds([0., 0., 0.], [10., 10., 10.])
+    >>> res = differential_evolution(rosen, bnds, workers=2, updating='deferred')
+
+Alternatively map-like callables can be supplied as a worker. Here the map-like function
```

Comment:
Nit: It'd be great to stress that the example below is semantically identical to the example above, and the only difference is the reuse of the pool, as discussed below.
On a quick read, _Alternatively_ made me think that the example that follows has some user-visible semantic differences.